src/app.config.js

```diff
@@ -43,4 +43,12 @@ config["slackInvitationLink"] = "https://bit.ly/in-core";
 config["slackWorkspaceLink"] = "https://in-core.slack.com";
 config["githubRelease"] = "https://github.com/IN-CORE/IN-CORE/releases/tag/";
 
-export default config;
+// Keycloak configuration
+config["keycloakConfig"] = {
+	// url: "http://localhost:8080/auth/", // for testing locally
+	url: `${hostname}/auth/`,
+	realm: "In-core",
```

Comment:
could you replace the relam with config["client_id"] so we don't need to hardcde the info twice?

Reply:
I don't think there is a variable for `In-core` I replaced it for client id as it was there...

Reply:
Ah right my bad I confused it with the "react-auth" client id